example/networking/mellanox.com_v1alpha1_macvlannetwork_ipam-whereabouts_cr.yaml

```diff
@@ -14,7 +14,7 @@
 apiVersion: mellanox.com/v1alpha1
 kind: MacvlanNetwork
 metadata:
-  name: example-macvlannetwork
+  name: example-macvlannetwork-ipam-whereabouts
```

Comment:
pod example need to be updated with the new network name or better yet just keep the original name for macvlan network both with and without ipam